lighthouse-core/config/default.js

```diff
@@ -22,15 +22,16 @@ module.exports = {
     "useThrottling": false,
     "gatherers": [
       "service-worker",
-      "offline"
+      "offline",
     ]
   },
   {
     "passName": "redirectPass",
     "useThrottling": false,
     "gatherers": [
       "http-redirect",
-      "html-without-javascript"
+      "html-without-javascript",
+      "start-url",
```

Comment:
Sadly it can not :( As I need the start url from the manifest and I can't retrieve the manifest unless the page loaded so if I go offline and the page can not be fetched I cannot get the manifest anymore.

Another option is to fail with a debugstring as I could not get the manifest?